converter/src/test/java/org/dswarm/converter/morph/RegexLookupTest.java

```diff
@@ -0,0 +1,72 @@
+/**
```

Comment:
y not moving this test to metafacture package? - that's where this new function comes from ;)